.github/workflows/bot.yml

```diff
@@ -13,94 +13,23 @@ env:
-            sparkProfile: "spark3.3"
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 8
-        uses: actions/setup-java@v2
-        with:
-          java-version: '8'
-          distribution: 'adopt'
-          architecture: x64
-      - name: Build Project
-        env:
-          SCALA_PROFILE: ${{ matrix.scalaProfile }}
-          SPARK_PROFILE: ${{ matrix.sparkProfile }}
-        run:
-          mvn clean install -D"$SCALA_PROFILE" -D"$SPARK_PROFILE" -pl hudi-examples/hudi-examples-spark,hudi-spark-datasource/hudi-spark -am -DskipTests=true $MVN_ARGS
-      - name: Quickstart Test
-        env:
-          SCALA_PROFILE: ${{ matrix.scalaProfile }}
-          SPARK_PROFILE: ${{ matrix.sparkProfile }}
-        run:
-          mvn test -Punit-tests -D"$SCALA_PROFILE" -D"$SPARK_PROFILE" -pl hudi-examples/hudi-examples-spark $MVN_ARGS
-      - name: UT - Common & Spark
-        env:
-          SCALA_PROFILE: ${{ matrix.scalaProfile }}
-          SPARK_PROFILE: ${{ matrix.sparkProfile }}
-        if: ${{ !endsWith(env.SPARK_PROFILE, '2.4') }} # skip test spark 2.4 as it's covered by Azure CI
-        run:
-          mvn test -Punit-tests -D"$SCALA_PROFILE" -D"$SPARK_PROFILE" -pl hudi-common,hudi-spark-datasource/hudi-spark $MVN_ARGS
-
-  test-flink:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        include:
-          - flinkProfile: "flink1.13"
-          - flinkProfile: "flink1.14"
-          - flinkProfile: "flink1.15"
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 8
-        uses: actions/setup-java@v2
-        with:
-          java-version: '8'
-          distribution: 'adopt'
-          architecture: x64
-      - name: Build Project
-        env:
-          SCALA_PROFILE: 'scala-2.12'
-          FLINK_PROFILE: ${{ matrix.flinkProfile }}
-        run:
-          mvn clean install -D"$SCALA_PROFILE" -D"$FLINK_PROFILE" -pl hudi-examples/hudi-examples-flink -am -Davro.version=1.10.0 -DskipTests=true $MVN_ARGS
-      - name: Quickstart Test
-        env:
-          SCALA_PROFILE: 'scala-2.12'
-          FLINK_PROFILE: ${{ matrix.flinkProfile }}
-        run:
-          mvn test -Punit-tests -D"$SCALA_PROFILE" -D"$FLINK_PROFILE" -pl hudi-examples/hudi-examples-flink $MVN_ARGS
-
   validate-bundles:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         include:
-          - flinkProfile: 'flink1.15'
-            sparkProfile: 'spark3.3'
-          - flinkProfile: 'flink1.14'
-            sparkProfile: 'spark3.2'
+          - flinkProfile: 'flink1.13'
+            sparkProfile: 'spark'
+          - flinkProfile: 'flink1.13'
+            sparkProfile: 'spark2.4'
           - flinkProfile: 'flink1.13'
             sparkProfile: 'spark3.1'
+          - flinkProfile: 'flink1.14'
+            sparkProfile: 'spark3.2'
+          - flinkProfile: 'flink1.15'
+            sparkProfile: 'spark3.3'
+          - flinkProfile: 'flink1.15'
+            sparkProfile: 'spark3'
```

Comment:
variate this to control `SPARK_PROFILE` which controls what bundles to download for testing in packaging/bundle-validation/ci_run.sh